<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lbl_number">
     <property name="text">
      <string>Player Number</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="email">
         <property name="inputMethodHints">
          <set>Qt::ImhEmailCharactersOnly</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>User Name</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="user_name"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Bank Balance</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="balance">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>9999999999999.990234375000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="cb_country_code"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Phone Number</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="phone">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Captcha</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="lbl_captcha">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Re-Generate</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="captcha"/>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pass"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_show_hide_pass">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_register">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>email</tabstop>
  <tabstop>pass</tabstop>
  <tabstop>btn_show_hide_pass</tabstop>
  <tabstop>user_name</tabstop>
  <tabstop>balance</tabstop>
  <tabstop>cb_country_code</tabstop>
  <tabstop>phone</tabstop>
  <tabstop>captcha</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>btn_register</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_register</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>userRegister()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>captcha_regen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_show_hide_pass</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>show_hide_password()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balance</sender>
   <signal>editingFinished()</signal>
   <receiver>Login</receiver>
   <slot>on_spinBox_valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pass</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Login</receiver>
   <slot>pass_check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>userRegister()</slot>
  <slot>captcha_regen()</slot>
  <slot>show_hide_password()</slot>
  <slot>on_spinBox_valueChanged()</slot>
  <slot>pass_check()</slot>
 </slots>
</ui>
